app/jobs/response_csv_export_operation_job.rb

```diff
@@ -2,9 +2,10 @@
 
 # Operation for exporting response CSV.
 class ResponseCsvExportOperationJob < OperationJob
-  def perform(operation, search: nil)
+  def perform(operation, search: nil, export_options: nil)
     ability = Ability.new(user: operation.creator, mission: mission)
-    result = generate_csv(responses(ability, search))
+    options_to_splat = export_options&.symbolize_keys || {}
+    result = generate_csv(responses(ability, search), **options_to_splat)
```

Comment:
Hmm, the idea behind this was to allow `answer_processor` to accept a named param `long_text_behavior` instead of parsing `options`, which your suggestion still accomplishes 🎉 I cleaned this up, thanks.